optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/construction/vrp/VRPConstructionBenchmark.java

```diff
@@ -14,9 +14,6 @@ public VRPConstructionBenchmark() {
         super(Examples.VEHICLE_ROUTING);
     }
 
-    @Param({"FIRST_FIT", "FIRST_FIT_DECREASING"})
```

Comment:
List variable is incompatible with these construction heuristic algorithms. It only works with the default CH configuration (`<constructionHeuristic/>`).

Reply:
But I thought FIRST_FIT is actually a default for  constructionHeuristic